cfme/ansible/repositories.py

```diff
@@ -277,6 +291,10 @@ def refresh(self):
         view.flash.assert_no_error()
         view.flash.assert_message("Embedded Ansible refresh has been successfully initiated")
 
+    @property
```

Comment:
OPTIONAL: You can probably use cached_property here. I use it in automate as well.